pkg/argon2/blamka_amd64.s

```diff
@@ -199,7 +199,7 @@ TEXT ·mixBlocksSSE2(SB), 4, $0-32
 	MOVQ out+0(FP), DX
 	MOVQ a+8(FP), AX
 	MOVQ b+16(FP), BX
-	MOVQ a+24(FP), CX
+	MOVQ c+24(FP), CX
```

Comment:
Just I want to learn a bit about this fix, could you write a short summary on why a replaced by c resolves those warnings?

Reply:
`go vet` verifies whether the offsets match the func declaration:

```
// func mixBlocksSSE2(out, a, b, c *block)
```

Here `c` is at offset `24` not `8`.